app/scripts/controllers/incoming-transactions.js

```diff
@@ -19,6 +19,9 @@ const networkTypeToIdMap = {
   [KOVAN]: String(KOVAN_CODE),
   [MAINNET]: String(MAINNET_CODE),
 }
+const fetch = newFetch({
```

Comment:
The purpose of this is to prevent timeout errors?

Reply:
Yes